storybook/src/components/ErrorMessage/ErrorMessage.docs.mdx

```diff
@@ -0,0 +1,19 @@
+import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as ErrorMessageStories from "./ErrorMessage.stories.tsx";
+import README from "../../../../packages/css/src/components/error-message/README.md?raw";
+
+<Meta of={ErrorMessageStories} />
+
+<Markdown>{README}</Markdown>
+
+<Primary />
+
+<Controls />
+
+## With a custom prefix
+
+Error messages are automatically prefixed with a visually hidden text, the Dutch phrase "Invoerfout: ".
```

Comment:
Why is visually hidden? This information could be relevant for anyone. Colour blind people might not use a screen reader, but having more explicit information that an there is an issue with their input could be helpful.

Reply:
True. This is usually combined with an icon for visual users, but we haven't made a decision about that yet